index copy.html

```diff
@@ -0,0 +1,298 @@
+            <a href="#site-project" class="link-portfolio">Portfolio</a>
+            <a href="#get-started" class="link-contact">Contact</a>
+            <img src="img/Group%2098.svg" alt="Background art" class="top-art">
+            <img src="img/Group%2051.svg" alt="Background art" class="group-51">
+            <img src="img/Group%2050.svg" alt="Background art" class="group-50">
+            <div class="bottom-art-1"></div>
+            <div class="bottom-art-2"></div>
+            <div class="bottom-art-3"></div>
+            <div class="bottom-art-4"></div>
+            <div class="bottom-art-6"></div>
+            <div class="bottom-art-5"></div>
+            <div class="bottom-art-7"></div>
+        </div>
+    </nav>
+</header>
+    <main class="flex-container">
+        <div>
+            <div class="name fade">
+                <h1>Felix Odete</h1>
+            </div>
+            <div class="about-me flex-container">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of
+                    my works.
+                    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+            </div>
+                <button type="button" class="btn-about button-wiggle">Start collaboration</button>
+            <div class="middle-image"></div>
+            <div class="side-image"></div>
+            <div class="image-geometry-2"></div>
+            <div class="image-geometry-5"></div>
+            <div class="image-geometry-4"></div>
+            <div class="image-geometry-3"></div>
+            <div class="rectangle-25-1"></div>
+            <div class="image-teal"></div>
+            <div class="projects-dots"></div>
+            <div class="name-dots"></div>
+        </div>
+        <div>
+            <h2 class="projects" id="site-project">Projects</h2>
+            <div class="computer-image"></div>
+            <div class="large-computer-image"></div>
+            <div class="project-section">
+                <h2>Project name goes here</h2>
+            </div>
+            <div class="grid-container">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
+                <div class="ruby-rails-rectangle">
+                    <p>Ruby on Rails</p>
+                </div>
+                <div class="javascript-rectangle">
+                    <p>Javascript</p>
+                </div>
+                <button data-modal-target="#modal" type="button" class="see-project">See this project</button>
+                <div class="arrow"></div>
+            </div>
+            <div class="image-geometry-4"></div>
+            <div class="macbook-image"></div>
+            <div class="project-section-1">
+                <h2>Project name goes here</h2>
+            </div>
+            <div class="grid-container-1">
```

Comment:
- [ ] It is required that the projects section should be created dynamically with javascript i.e all the HTML of the projects section is created when the page loads. Kindly remove all the HTML code of the section, and dynamically render the content using javascript. Create an array of objects to store the information for each project and render them dynamically.
The array of objects for the project cards should be something like this
``` javascript
const project = [
                            {project card 1 information},
                            {project card 2 information},
                            {project card 3 information},
                            ...
                          ]
```